client/src/components/Review/ReviewPanel.js

```diff
@@ -28,53 +30,82 @@ const Header = styled.div`
   font-weight: bold;
   
   color: #FBD702;
-
   width: 100%;
   margin-bottom: 15px;
 `
 const STYLED_form = styled.form`
   display: flex;
   flex-direction: column;
-
   padding: 15px;
   margin: 5px 0 5px 0;
-
   .buttonContainer {
     display: flex;
     justify-content: center;
   }
-
   border-radius: 10px 10px 10px 10px;
   background-color: white;
-`
-// COMPONENT
-class ReviewPanel extends Component {
+
+// import Firebase from './firebase';
+const buttonStyle = {
+  margin: "10px 10px 10px 10px"
+};`
+
+
+class ReviewPanel1 extends Component {
   constructor(props) {
     super(props);
 
-  // STATE
+    // STATE
     this.state = {
       newUser: {
-        user_id: '',
-        rating: '',
-        internet_rating: '',
+        user_id: null,
+        rating: null,
+        internet_rating: null,
         comments: ''
       },
-      user_id: ["8","4","2"],
       rating: ["1", "2", "3"],
       internet_rating: ["1", "2", "3"],
+      uid: this.props.firebase.auth.currentUser.uid
     };
-    
+
     this.handleTextArea = this.handleTextArea.bind(this);
     this.handleFormSubmit = this.handleFormSubmit.bind(this);
     this.handleClearForm = this.handleClearForm.bind(this);
     this.handleInput = this.handleInput.bind(this);
   }
 
+
+  // COMPONENT
+  componentDidMount() {
+    axios
+      .get(`https://wheretocode-master.herokuapp.com/users/${this.state.uid}`)
+      .then(user => {
+        console.log("get", user);
+        console.log("userid", user.data[0].id);
+        let currentUserId = {
+          user_id: user.data[0].id,
+          rating: null,
+          internet_rating: null,
+          comments: ''
+        }
+        this.setState({
+          newUser: currentUserId
+        })
+      }
+
+      )
+      .catch(error => {
+        console.log(error);
+      })
+
+  }
+
   // METHODS
   handleInput(e) {
     let value = e.target.value;
     let name = e.target.name;
```

Comment:
what is the point of this? Would be less work to just say [e.target.name]: e.target.value